src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -93,5 +93,12 @@ def __call__(self, path, **params):
 class RgwBucket(RESTController):
 
     def create(self, bucket, uid):
-        rgw_client = RgwClient.instance(uid)
-        return rgw_client.create_bucket(bucket)
+        try:
+            rgw_client = RgwClient.instance(uid)
+            return rgw_client.create_bucket(bucket)
+        except Exception as e:  # pylint: disable=broad-except
```

Comment:
The problem here, and also in the `RgwProxy` controller, is that you are catching ALL errors, and you are skipping our current exception handler that prints the stack trace in the log file.

Can you be a bit more precise/specific on the kind of exceptions you want to catch here.